webcomponents/tool/src/main/frontend/js/calendar/sakai-calendar.js

```diff
@@ -162,16 +169,14 @@ export class SakaiCalendar extends LionCalendar {
 
     return html`
 
+      <div>${this.i18n.days_message.replace("{}", this.days)}</div>
```

Comment:
I still dislike this a lot but anyway.

Reply:
I don't want to mess with the i18n stuff to enable implicit loading of the props. I don't want to destabilise it.

Reply:
I can understand that, but then it feels like the 'i18n stuff' is flawed from design, especially considering the options we used to have.

Reply:
Can't you make it just like in Rubrics? There's this available method:

`function tr(namespace, key, options) {`

Reply:
Yes, that is the alternative. But, you still have to load the properties first in a loadProperties call. That's what I meant by implicit loading in the tr call. IE: in the tr body, if the namespace isn't loaded, load it. I'm worried about messing with that when I can just replace it like this.

Reply:
I think that should be something that is investigated for 23, but not for this change which will need to be backported. I don't want to make anything break.

Reply:
But you are right :)